File: pset6/mario.py
import cs50

height = input("height: ")

while height.isnumeric() == False:
    height = input("height: ")

height = int(height)

while height < 1 or height > 8:
    height = int(input("height: "))

for i in range(height):
    for j in range(height * 2 + 2):
        if (j + 1) < (height - i) or ((j + 1) <= height + 2 and (j + 1) > height):
            if (j + 1) > height + 3 + i:
                continue
            else:
                print(" ", end="")
        else:
            if ((j + 1) <= height and (j + 1) >= height - i) or ((j + 1) > height + 2 and (j + 1) <= height + 3 + i):
                print("#", end="")
    print("")

File: pset1/hello/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int x;
    do
    {
        x = get_int("Height: ");
    }
    while (1 > x || x > 8);

    for (int l = 0; l < x; l++ )
    {
        for (int c = 1; c <= ((x * 2) + 2); c++ )
        {
           if (c < (x - l) || (c <= (x + 2) && c > x) )
           {
               if (c > (x + 3 + l))
               {
                   continue;
               }
               else
               {
                   printf(" ");
               }
           }
           else
           {
               if ((c <= x && c >= (x - l)) || (c > (x + 2) && c <= (x + 3 + l)))
               {
                   printf("#");
               }
           }
        }
        printf("\n");
    }
}